charts/opensearch/values.yaml

```diff
@@ -104,6 +104,10 @@ config:
 extraEnvs: []
 #  - name: MY_ENVIRONMENT_VAR
 #    value: the_value_goes_here
+# 2.12 onwards a custom strong password needs to be provided in order to setup demo admin user.
+# Cluster will not spin-up without this unless demo config install is disabled.
+#  - name: OPENSEARCH_INITIAL_ADMIN_PASSWORD
```

Comment:
Can we at least output some error msg for people to use to follow?
Instead of failing out of nowhere and need to check readme?

Reply:
I will need your help adding that.

Reply:
After discussing with @DarshitChanpura and @derek-ho seems like we cannot do too much on the log, as kubectl will show pods status as ERROR, and user can just check kubectl logs for the exact comment here:
https://github.com/opensearch-project/security/blob/c06365ca94c9a1d15e85b578a1ae48168bf0bca7/src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java#L155-L157

However, I wonder if we can change the commend of helm install, to reflect this env var requirement directly in the commend, so people are aware that we need to set such env var in order to deploy the pods successfully.
https://github.com/opensearch-project/security/blob/c06365ca94c9a1d15e85b578a1ae48168bf0bca7/src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java#L155-L157

Reply:
That is not an option here, since we'll have to pass a default plain-text password in values.yml anyway if we want to allow users to set via --set. We are trying to move away from default passwords and hence this is not be the best path forward.